net/dnsfallback/dns-fallback-servers.json

```diff
@@ -6,25 +6,33 @@
 			"RegionName": "r1",
 			"Nodes": [
 				{
-					"Name": "1c",
+					"Name": "1f",
```

Comment:
revert this?

Reply:
👍🏼   https://github.com/tailscale/tailscale/pull/9602